pkg/controllers/nodeclass/status/controller.go

```diff
@@ -52,10 +53,11 @@ type Controller struct {
 	subnet          *Subnet
 	securitygroup   *SecurityGroup
 	readiness       *Readiness //TODO : Remove this when we have sub status conditions
+	tags            *Tags
```

Comment:
Why is this just tags? It should be checking more than tags -- should be checking the overall viability of the calls